pkg/compression/compression.go

```diff
@@ -9,13 +9,11 @@
 package compression
 
 import (
-	"flag"
+	"os/exec"
 
 	"github.com/linuxboot/fiano/pkg/guid"
 )
 
-var xzPath = flag.String("xzPath", "", "Path to system xz command used for lzma encoding. If unset, an internal lzma implementation is used.")
```

Comment:
I think it would still be good to have this option, but have it default to "xz".

Reply:
+1

The rest looks good by the way ! Thanks for adding a test !

Reply:
Done! I kept the LookPath change, figured it wouldn't hurt :)